src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +82,21 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    public List<Person> sortedListView() {
+        List<Person> sortedList = new ArrayList<>();
+        for (Person p : internalList) {
+            sortedList.add(p);
+        }
+        Collections.sort(sortedList, new Comparator<Person>() {
+            @Override
+            public int compare(Person p1, Person p2) {
+                String person1Name = p1.getName().fullName;
+                String person2Name = p2.getName().fullName;
```

Comment:
Also, it is better to use toString() method to retrieve the full name instead of calling the fullName attribute.